### DISCUSSION TITLE
Is there any control flag can disable output info?

### DISCUSSION THREAD

**Discussion by leizhenyuan** [pypa/setuptools#3988] [Q&A] [answered]
![image](https://github.com/pypa/setuptools/assets/110007460/0d52dbca-039a-4b4d-8907-391f5e109486)
I believe this message are print by setup tools, is there any flag or env variable can mute them ? thanks

**Comment by abravalheri** [comment, marked as answer]
If you are using `python -m build` you can reduce the verbosity with `python -m build -C verbose=0` (it will not remove the output completely, but reduce at least).

If you are using `setuptools` programmatically you can set the level of the root logger (standard Python `logging` system).

**Reply by leizhenyuan** [reply]
Works well for me, thanks for reply
